src/app/backing/components/TotalBackingMetric/TotalBackingMetric.tsx

```diff
@@ -1,47 +1,10 @@
-import { Metric } from '@/components/Metric/Metric'
-import { Typography } from '@/components/TypographyNew/Typography'
-import { TokenImage } from '@/components/TokenImage'
 import { stRIF } from '@/lib/constants'
+import { TokenAmountDisplay } from '@/components/TokenAmountDisplay'
 
 interface TotalBackingMetricProps {
   totalBacking: string
 }
 
 export const TotalBackingMetric = ({ totalBacking }: TotalBackingMetricProps) => {
```

Comment:
Im not sure if we should have some many components.

Reply:
What do you mean by "so many components"?

Reply:
Now this is just a wrapper of tokenAmountDisplay

Reply:
for now let's keep it